client/components/push-notifications.jsx

```diff
@@ -171,7 +171,7 @@ export default class PushNotifications extends React.Component {
 
       <h3>Push Notification Subscription</h3>
       <code>
-        {JSON.stringify(this.state.subscription, null, 4)}
+        {JSON.stringify(this.state.subscription)}
```

Comment:
Nope. It generates the curl command. You would mostly copy paste it in the terminal to send a push notification.